src/resolvers.ts

```diff
@@ -212,6 +213,66 @@ const resolvers = {
             return createSubmissionResult.data
         },
 
+        moderationPanelUpdateSubmission: async (_parent: unknown, args: {
+            input: gqlType.ModerationAutofillDatabaseSubmissionInput
+        }): Promise<gqlType.Submission> => {
+            const submissionLocationInformation: gqlType.UpdateSubmissionInput = {
+                facility: {
+                    contact: {  
+                        phone: '', 
+                        website: null,
+                        googleMapsUrl: '',
+                        address: {
+                            postalCode: '',
+                            prefectureEn: '',
+                            addressLine1En: '',
+                            addressLine2En: '',
+                            cityEn: '',
+                            prefectureJa: '',
+                            addressLine1Ja: '',
+                            addressLine2Ja: '',
+                            cityJa: ''
+                        }
+                    },
+                    nameEn: '',
+                    nameJa: '',
+                    mapLatitude: 0,
+                    mapLongitude: 0,
+                    healthcareProfessionalIds: []
+                }             
+            }
+
+            const autofillPlacesInformation
```

Comment:
this looks like logic that should be moved to the service

Reply:
It is there now!